modules/data-catalog-tag/README.md

```diff
@@ -109,15 +128,20 @@ parent: projects/project-data-product/datasets/exposure
 location: europe-west1
 template: projects/project-datagov/locations/europe-west1/tagTemplates/test
 fields:
-  owner_email: example@example.com
+  owner_email: 
```

Comment:
Can we have a test that also covers display name here?

Reply:
Display name is an output of the resource. Removed in the main.tf, no need to add it here.